_ont_xgs/ont-hisense-ltf7267-bha+.md

```diff
@@ -12,14 +12,377 @@ parent: Hisense
+    |  15 | 4096 |  15 | 4096 |No filter| 0 |  15 | 4096 |   0 |    1 |
+    |  15 | 4096 |   8 |  835 |No filter| 1 |  15 |    0 |   8 |  100 |
+    |-----|------|-----|------|---------|---|-----|------|-----|------|
+```
+
+## Changing the Hardware Version
+
+```sh
+# vi /config/scfg.txt
+```
+
+Append line below to the file and save it to change HWVER
+
+```
+CHAR-ARRAY      CFG_ID_GPON_VERSION = {0x46,0x35,0x36,0x38, 0x34,0x53,0x5f,0x76, 0x31,0x00,0x00,0x00, 0x00,0x00}; ##GPON version string, default value is V1.0
+```
+
+Reboot ONT to apply the change
+
+## Changing the Equipment ID
+
+```sh
+# vi /config/scfg.txt
+```
+
+Append line below to the file and save it to change EQID
+
+```
+CHAR-ARRAY      CFG_ID_GPON_EQID = {0x46,0x49,0x42,0x45, 0x52,0x20,0x42,0x6f, 0x78,0x00,0x00,0x00, 0x00,0x00,0x00,0x00, 0x00,0x00,0x00,0x00}; ##GPON ME ONU2G equiment id value, default is saturn
+```
+
+Reboot ONT to apply the change
+
+## Changing Serial Number
+
+```sh
+# vi /config/scfg.txt
+```
+
+Append lines below to the file and save it to change Serial Number
+
+```
+STRING CFG_ID_PON_VENDOR_ID = SMBS;
+INT CFG_ID_PON_VSSN = 0xAABBCCDD;
+```
+
+Reboot ONT to apply the change
+
+## Changing OLT Emulation Type
+
+```sh
+# vi /config/scfg.txt
+```
+
+Append line below to the file and save it to change Serial Number
+
+```
+STRING          CFG_ID_PON_OLT_TYPE                             = ALCL; ##GPON OLT Vendor name, support ZTE,ADTRAN,ALCL,CALIX,SUMITOMO,CORTINA,HUAWEI
+
+```
+
+Reboot ONT to apply the change
+
+## Changing OMCC Version
+
+{% include alert.html content="In Italy, if you are under Huawei OLT it's mandatory to use 0xA3, while on Alcatel 0xB4, otherwise you will get O5 status but no MIBs" alert="Warning" icon="svg-warning" color="red" %}
+
+
+```sh
+# vi /config/scfg.txt
+```
+
+Append line below to the file and save it to change Serial Number
+
+```
+CHAR            CFG_ID_OMCC_VERSION                             = 0xB2;
+```
+
+Reboot ONT to apply the change
+
+## Changing LOID Username/Password
+
+{% include alert.html content="The value 0x0 is null, take not of your LoID and password from original ONT" color="red" %}
+
+
+```sh
+# vi /config/scfg.txt
+```
+
+Append line below to the file and save it to change Serial Number
+
+```
+CHAR-ARRAY      CFG_ID_LOID                                                         = {0x30,0x30,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0};
+CHAR-ARRAY      CFG_ID_PASSWD                                                       = {0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0};
+```
+
+Reboot ONT to apply the change
+
+## Spoofing firmware version
+
+```sh
+# fw_setenv img_version0 20220527052622
+# fw_setenv img_version0 20220527052622
+```
+
+Reboot ONT to apply the change
+
+# Known Bugs
+- `ALCL` OLT mode use some static configuration on MIBs, so if your OLT has strict configuration, it might not work
```

Comment:
```suggestion
- `ALCL` OLT mode uses some static configurations on MIBs, so if your OLT has strict configuration checks it might not work properly
```